debian/control

```diff
@@ -0,0 +1,165 @@
+Source: letsencrypt
+Section: web
+Priority: optional
+Maintainer: Debian letsencrypt <letsencrypt-devel@lists.alioth.debian.org>
+Uploaders: Harlan Lieberman-Berg <hlieberman@setec.io>,
+	   Francois Marier <francois@debian.org>
+Build-Depends: debhelper (>= 9),
+	       dialog,
+               dh-python,
+               python-all,
+	       python-augeas,
+               python-configargparse,
+	       python-configobj,
+               python-dialog,
+               python-docutils,
+	       python-mock,
+	       python-ndg-httpsclient,
+               python-nose,
+               python-nosexcover,
+               python-openssl (>= 0.15),
+               python-parsedatetime,
+	       python-psutil,
+	       python-pyasn1,
+	       python-pyparsing,
+               python-repoze.sphinx.autointerface,
+	       python-requests,
+               python-rfc3339,
+	       python-setuptools,
+               python-sphinx,
+               python-sphinx-rtd-theme,
+       	       python-tz,
+               python-werkzeug,
+               python-zope.component,
+	       python-zope.interface
+Standards-Version: 3.9.6
+Homepage: https://www.letsencrypt.org/
+X-Python-Version: >= 2.7
+
+Package: python-acme
+Section: python
+Architecture: all
+Depends: openssl,
```

Comment:
I don't think openssl is required.

However, `ca-certificates` is required for communication with Let's Encrypt CA (might be put as dep in the core package).